pkgs/development/libraries/sofia-sip/default.nix

```diff
@@ -1,19 +1,22 @@
-{ stdenv, fetchurl, glib, openssl, pkgconfig }:
+{ stdenv, fetchFromGitHub, glib, openssl, pkgconfig, autoreconfHook }:
 
 stdenv.mkDerivation rec {
-  name = "sofia-sip-1.12.11";
+  pname = "sofia-sip";
+  version = "1.13";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/sofia-sip/${name}.tar.gz";
-    sha256 = "10bwsdfijpbk9ahlfpk94kzdapxiahl9mljpgwghvq1630pbq09b";
+  src = fetchFromGitHub {
+    owner = "freeswitch";
+    repo = pname;
+    rev = "f6f29b483e9c31ce8d3e87419ec3deea8679312d"; # upstream does not seem to believe in tags
+    sha256 = "sha256-KD3sPrf9GjA357jomFL8LtZ3rp25v3Vy8QfSZNSozso=";
   };
 
   buildInputs = [ glib openssl ];
-  nativeBuildInputs = [ pkgconfig ];
+  nativeBuildInputs = [ autoreconfHook pkgconfig ];
 
   meta = with stdenv.lib; {
     description = "Open-source SIP User-Agent library, compliant with the IETF RFC3261 specification";
-    homepage = "http://sofia-sip.sourceforge.net/";
+    homepage = "https://github.com/freeswitch/sofia-sip";
     platforms = platforms.linux;
```

Comment:
```suggestion
    platforms = platforms.unix;
```

Reply:
This does build on macos 10.14 _if_ you provide it with the `buildInput` `darwin.apple_sdk.frameworks.SystemConfiguration`

Reply:
We do have quite a bit of stuff with platform-conditional build inputs.